### PR TITLE
Show the Orginal Timezone

### LINKED ISSUE #1640: Timezones should match location of where event occurred, not viewer's location

### PR DESCRIPTION
**Pull request by JNygaard-Skylight**
# PULL REQUEST

## Summary
Datetimes created in the FHIR conversion process include time zone information. This PR removes code that would have converted datetimes into the browser's local time and instead shows the date time as is, including the the UTC offset.

## Related Issue
Fixes #1640 

## Additional Information
Anything else the review team should know?

## Checklist

- [ ] If this code affects the other scrum team, have they been notified? (In Slack, as reviewers, etc.)

[//]: # (PR title: Remember to name your PR descriptively!)

### CODE REVIEW

**Comment by JNygaard-Skylight** [comment]
@sarahtress This is ready for design review: 
image
image

File: containers/ecr-viewer/src/app/format-service.tsx [hunk 1]
```diff
@@ -76,32 +76,34 @@ export const formatAddress = (
...
+
+    // We only need HH:MM from the time
+    const [hours, minutes] = time.split(":");
+
+    // Convert 24-hour time to 12-hour time
+    const hoursInt = parseInt(hours, 10);
+    const suffix = hoursInt >= 12 ? "PM" : "AM";
+    const hours12 = ((hoursInt + 11) % 12) + 1; // Convert 24h to 12h format
+
+    return `${month}/${day}/${year} ${hours12}:${minutes} ${suffix} ${timeZone === undefined ? "UTC" : timeZone}`;
```
Comment (nickbristow) [inline containers/ecr-viewer/src/app/format-service.tsx:102]: you can rewrite the second part as
```${timeZone || "UTC"}```

File: containers/ecr-viewer/src/app/view-data/components/tests/__snapshots__/LabInfo.test.tsx.snap [hunk 2]
```diff
@@ -976,7 +976,7 @@ Burbank, CA
                     <div
                       class="grid-col-auto maxw7 text-pre-line lab-text-content"
                     >
-                      09/28/2022 5:00 PM EDT
+                      undefined/undefined/09/28/2022 2:00 PM PD
```
Comment (nickbristow) [inline containers/ecr-viewer/src/app/view-data/components/tests/__snapshots__/LabInfo.test.tsx.snap:979]: Whats this undefined? Is that intentional?

**Review by nickbristow — CHANGES_REQUESTED** [review]

File: containers/ecr-viewer/src/app/format-service.tsx [hunk 3]
```diff
@@ -76,32 +76,34 @@ export const formatAddress = (
...
-    month: "2-digit",
-    day: "2-digit",
-    timeZoneName: hasTime ? "short" : undefined,
-  };
-  if (hasTime) {
-    options.hour = "numeric";
-    options.minute = "2-digit";
-  } else {
-    options.timeZone = "UTC"; // UTC, otherwise will have timezone issues
+export const formatDateTime = (dateTimeString: string): string => {
```
Comment (BobanL) [inline containers/ecr-viewer/src/app/format-service.tsx:79]: Can you write a jsdoc for this function and include the example format you expect dateTimeString to be in?

**Review by nickbristow — APPROVED** [review]

**Pull request merged**